pyproject.toml

```diff
@@ -33,6 +33,9 @@ zip-safe = false
 [tool.setuptools.dynamic]
 version = { attr = "xmipy.__version__" }
 
+[tool.setuptools.packages.find]
+include = ["xmipy", "xmipy.*"]
```

Comment:
This is re-added, otherwise "tests" would be added to the .whl package, since it is now a findable module.

Reply:
I see, thanks for fixing that.
I removed it since with `packages.find` submodules were not found.
I guess, I should have added "packagename.*" to fix that.

Reply:
This now revised with [`tool.hatch.build.targets.wheel.packages`](https://hatch.pypa.io/latest/config/build/#packages)